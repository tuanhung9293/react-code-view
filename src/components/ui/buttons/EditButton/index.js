import React from 'react'
import IconButton from '../IconButton'
import edit from 'assets/edit.svg'

function EditButton(props) {
  return <IconButton {...props}><img src={edit} alt="Edit" /></IconButton>
}

export default EditButton