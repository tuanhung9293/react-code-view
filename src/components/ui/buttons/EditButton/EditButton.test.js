import React from 'react'
import {shallow} from 'enzyme'
import EditButton from './'

it('renders as a EditButton', () => {
  expect(shallow(
    <EditButton />
  )).toMatchSnapshot()
})

it('should EditButton pass props', () => {
  expect(shallow(
    <EditButton dataTest="dataTest" />
  )).toMatchSnapshot()
})
