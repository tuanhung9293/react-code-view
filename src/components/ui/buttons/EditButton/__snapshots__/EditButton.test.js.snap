// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as a EditButton 1`] = `
<IconButton>
  <img
    alt="Edit"
    src="test-file-stub"
  />
</IconButton>
`;

exports[`should EditButton pass props 1`] = `
<IconButton
  dataTest="dataTest"
>
  <img
    alt="Edit"
    src="test-file-stub"
  />
</IconButton>
`;
